src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/AssemblyResourceManager.cs

```diff
@@ -56,31 +56,23 @@ public override bool TryInvokeMember(InvokeMemberBinder binder, object[] args, o
 
         private bool TryGetResourceValue(string resourceName, object[] args, out object result)
         {
-            if (PlatformDetection.IsNetNative)
-            {
-                result = string.Empty;
-                return true;
-            }
-            else
-            {
 #if netcoreapp
-                var type = _resourceAssembly.GetType("System.SR");
+            Type type = _resourceAssembly.GetType("System.SR");
```

Comment:
There are many places it'd simplify code access to resources if they had a unified name between the codebases. It was something I wanted to ask if there was a way to implement and which name would be better.

Reply:
The only reason we keep it different is for localization, since NetFx resource strings are localized while netcore aren't.

Reply:
If netcore doesn't because it won't be localized care could we just use the netfx name for both projects? In theory then core localization would happen for free because the resources should end up interchangeable.

Reply:
I was thinking/wondering the same thing...

Reply:
I can try...

Reply:
Definitely a separate PR for the future. We'll see who gets around to it first now that we've agreed that it should be done.